TCMSApp/src/client/app/defects/defects.html

```diff
@@ -72,7 +75,21 @@
                                 <td>
                                     {{defect.priority}}
                                 </td>
-
+                                <td>
+                                    <select ng-init ="defect.status == undefined?defect.status = 'notFix':defect.status"
+                                            ng-class="{'btn btn-danger sizeStatus': defect.status == 'notFix',
```

Comment:
btn and sizeStatus does not change. wee can extrtact them to regular class attribute. Or you had some ideas to do like that?